linode/manager
# M3-5896: Enable Cypress recording and parallelization via Jenkins

## Description 📝

**What does this PR do?**
This implements the Cloud-specific changes needed to enable parallelization when running our Cypress tests via Jenkins.

Specifically, this makes the following changes:

- Defines additional services in `docker-compose.yml` to facilitate running multiple Cypress test runners at once
- Refactors `docker-compose.yml` to reduce redundancy
- Passes build info to the test runner containers via environment variables

## How to test 🧪
There's no real testing that you can do locally to verify these changes, but feel free to reach out via Slack and I can refer you to successful builds which use this code.
